docs/diagrams/RelateActivityDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+
+
+title Activity: Relate command
+start
+:User executes relate command;
+
+if () then ([else])
+    :Display error message;
+else ([two numeric IDs provided])
+    if () then ([else])
+        :Display error message;
```

Comment:
Just a nit but can change from `Display error message` into `Show error message`

So match the same style as what are existing, referring down below.
![image](https://github.com/AY2324S2-CS2103T-F12-1/tp/assets/110763159/76c0d6da-c35c-4ffe-b7c7-580f7d8d7cf0)